### ISSUE TITLE
MLS Data

### ISSUE THREAD

**Issue by subotic** [dasch-swiss/0807-mls-app-web#2]
Here is the download link to the data: https://www.dropbox.com/s/rybb8weli3im1ao/mls-2019-03-25.zip?dl=1

Here is a small Diagram showing the relationships of the data:

[MLS-Schema.pdf](https://github.com/dasch-swiss/0807-mls-app-web/files/3004196/MLS-Schema.pdf)

And here is the current Home-Page:

![Screenshot 2019-03-25 16 48 42](https://user-images.githubusercontent.com/400790/54933980-fedc3100-4f1d-11e9-848c-0f5708cd231f.png)

Lexika view:

![Screenshot 2019-03-25 16 50 37](https://user-images.githubusercontent.com/400790/54934165-55496f80-4f1e-11e9-8f92-72a260f4a9ce.png)

Lemmata view:

![Screenshot 2019-03-25 16 49 16](https://user-images.githubusercontent.com/400790/54934140-4a8eda80-4f1e-11e9-8f0f-83120d9b0ded.png)

Artikel view:

![Screenshot 2019-03-25 16 51 05](https://user-images.githubusercontent.com/400790/54934201-63978b80-4f1e-11e9-86c9-8bd0980f8a56.png)

**Comment by subotic** [comment]
I've found a big bug in the data. I will post the updated data tomorrow.

**Comment by subotic** [comment]
Sorry for the holdup. I've run into additional problems. Here is the data: https://www.dropbox.com/s/kmugd2o70jta0jg/mls-2019-03-27.zip?dl=1

**Comment by subotic** [comment]
Here is the (hopefully) fixed data: https://www.dropbox.com/s/xubd3mmzqig85rt/mls.trig-2019-03-27_002.zip?dl=1

This time, I've exported all three graphs into one file. You only need to import this one file with the overwrite checkmark and you will need to add these graphs to the overwrite list:

- `http://www.knora.org/data/admin`
- `http://www.knora.org/data/0807/mls`
- `http://www.knora.org/ontology/0807/mls`

**Comment by flavens** [comment]
So this one file corresponds to the three files you gave to me previously?

**Comment by subotic** [comment]
correct

**Comment by flavens** [comment]
@subotic I cannot import the data (I tried it three times)
<img width="1624" alt="Screen Shot 2019-03-28 at 09 59 47" src="https://user-images.githubusercontent.com/29532462/55144054-4fcb6f80-5140-11e9-87df-da8cca29fbcd.png">

**Comment by subotic** [comment]
hmm, strange. This should have been caught during bulkimport. I will change the ontology and make a new export.

**Comment by subotic** [comment]
Here is a new export: https://www.dropbox.com/s/u6ovd2tsa0mkdh3/mls.trig-2019-03-28_001.zip?dl=1

You first need to delete the two mls graphs.

**Comment by subotic** [comment]
The permissions still allow only access for the root user. Didn't work as I have expected.

**Comment by flavens** [comment]
> Here is a new export: https://www.dropbox.com/s/u6ovd2tsa0mkdh3/mls.trig-2019-03-28_001.zip?dl=1
> 
> You first need to delete the two mls graphs.

what do you mean by "You first need to delete the two mls graphs.", is it mls-data and mls-onto in graphDB?

**Comment by subotic** [comment]
yes

**Comment by flavens** [comment]
@subotic still the same (I tried twice):
<img width="2032" alt="Screen Shot 2019-03-28 at 12 34 58" src="https://user-images.githubusercontent.com/29532462/55154788-fc641c00-5155-11e9-8467-48f2f37ab305.png">

**Comment by subotic** [comment]
ok, this is another property. I will change the ontology and make another import.

**Comment by subotic** [comment]
Here is a new export: https://www.dropbox.com/s/y0nsudbz2g8mxwv/mls.trig-2019-03-28_002.zip?dl=1

**Comment by subotic** [comment]
Here is the same data but with correct permissions, so that everyone can view the data:
https://www.dropbox.com/s/lrfyoybm3avpon8/mls.trig-2019-03-28_003.zip?dl=1

**Comment by flavens** [comment]
@subotic have you added the Exemplar in the onto and data?

**Comment by subotic** [comment]
no, the data you have has only Lemma, LemmaLocation, LemmaOccupation, Location, Occupation and Library.

I'm working on importing the rest of the properties for Lemma.

**Comment by flavens** [comment]
Yes, but I cannot even find Exemplar in the ontology, whereas there are Article and Lexika (but no data yet).

**Comment by subotic** [comment]
No, it is not in the ontology yet. I do it in parallel, add to the ontology as I ad data.
